R/pkg/R/functions.R

```diff
@@ -132,6 +132,27 @@ NULL
 #' df <- createDataFrame(as.data.frame(Titanic, stringsAsFactors = FALSE))}
 NULL
 
+#' Miscellaneous functions for Column operations
+#'
+#' Miscellaneous functions defined for \code{Column}.
+#'
+#' @param x Column to compute on. In \code{sha2}, it is one of 224, 256, 384, or 512.
+#' @param y Column to compute on.
```

Comment:
I think roxygen automatically chooses the order of the arguments based on the order they appear in the file, and ignores the order we specify. So even if I move `y` before `x` here, in the generated doc, `x` will still appear before `y`. Indeed, as you can see from the screenshot, `...` appears before `y`.